pkgs/development/compilers/rust/rustc.nix

```diff
@@ -141,6 +141,12 @@ in stdenv.mkDerivation rec {
     python x.py dist rustc-dev
     tar xf build/dist/rustc-dev*tar.gz
     cp -r rustc-dev*/rustc-dev*/lib/* $out/lib/
+    rm $out/lib/rustlib/install.log
+    for m in $out/lib/rustlib/manifest-rust*
+    do
+      mv $m .
+      sort $(basename $m)> $out/lib/rustlib/$(basename $m)
```

Comment:
```suggestion
      sort --output=$m > $m
```

Reply:
Might be also interesting to bring up this fact upstream.

Reply:
Will this work even if the file size exceeds the size of the in memory buffer that `sort` allocates?

Reply:
AFAIK even when reading from a pipe `sort` has a buffer on the order of 'tens of megabytes' and the manifests are only a few kilobytes, so it seems fairly safe.

Reply:
I think coreutils/sort will create temporary files and allocate is sort buffer according to available main memory: https://github.com/coreutils/coreutils/blob/master/src/sort.c#L1462

Reply:
But this should never matter because the linking process of rustc will use up way more RAM than the file listing will take.